src/domains/integration/dto/index.ts

```diff
@@ -221,15 +223,6 @@ export class AuthorizationRequestDTO {
   @IsValidOrganization()
   @IsNotEmpty()
   readonly organizationName!: string;
-
```

Comment:
Please modify Swagger documentation and let frontend team know they have to modify  /api/integration/:issueProviderName/authorization request body.